_posts/2019-09-30-team-working-agreements.md

```diff
@@ -0,0 +1,261 @@
+
+<aside class="dialogue">
+  <div class="answer">
+    <img src="/images/dialogue/steve.jpg">
+    <p class="intro">Steve Hicks says...</p>
+    <p>
+      During my first team agreement brainstorm, I felt like we spent a lot of time talking about details 
+      related to specific incidents. On my next team, I made a strong effort to constrain the suggestions 
+      to avoid this. To me, team values were really important for this new team - and I did such a good 
+      job of encouraging those types of suggestions that it's pretty much all we ended up with.
+    </p>
+    <p>
+      While this made me feel like I understood the values of my team better, we're months later _still_ 
+      paying for this bias. Our working agreement lacked important procedural and tactical definition. We've 
+      been adding things like JIRA hygiene and responsibilities of project leaders as they come up, but I would
+      not push a working agreement brainstorm so strongly in one direction again.
+    </p>
+  </div>
+</aside>
+
+(ADAM: Add an Adam says... section to share the habits that the Galleries team agreed upon)
+
+Remember that the working agreement is a living document. If the team learns that it missed something in the
+brainstorm, don't hesitate to update the agreement.
+
+### Working agreements are hard to define for teams without a well-established identity.
+
+For a team that has a strong and cohesive identity, the agreement is likely to affirm many things the team is
+already believing and doing. For a less cohesive team the agreement is harder to pin down. Many opinions will
+surface, and they may be in conflict with each other.
+
+While this is indeed challenging, it is also important to note that teams that lack identity value _greatly_ from
+the team agreement brainstorm. Provided there is
+[psychological safety](https://github.com/artsy/README/blob/master/culture/engineering-principles.md#psychological-safety),
+this is a really great time for the team to learn about what matters to each other, and move toward establishing an
+identity.
+
+### Facilitating the brainstorm can be difficult on a distributed team.
+
+Some of our teams were able to brainstorm with everyone on-site. They were able to rely on post-it notes and
+in-person conversations, and read each other's non-verbal communication. Other teams needed to accommodate remote
+members, and used tools like [Miro](http://miro.com/) to facilitate. This _can_ present a challenge...but if you're
+already a distributed team, they are likely the same types of challenges you've worked to overcome for _all_
+meetings.
+
+The differences are worth noting, though. The importance of reading each other's non-verbal communication is
+magnified when you're having conversations about identity. It's quicker to organize and re-organize real-life
+post-its than virtual post-its in an online tool. These are all opportunities to get better at being a distributed
+team.
```

Comment:
Good points. There are some "hacks" for ensuring a level playing-field for remote and in-person team members, such as:
* Everyone conduct the meeting remotely.
* At each meeting stage, ask each participant for input explicitly rather than depend on them chiming in.